pages/glossary/start-end-content.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Start and end of the content
+key: start-end-content
+unambiguous: true
+objective: true
+input_aspects:
+  - Accessibility tree
+  - CSS styling
+  - DOM tree
+---
+
+A node is _at the start_ of a [block of content][] if one of the following is true:
+
+- the node is the first [perceivable content][] inside this [block][]; or
+- the is before the first [perceivable content][] in this [block][] and there is no other [perceivable content][] between the node and the [block][].
+
+In other words, the node is either the first [perceivable content][] inside the [block][], any non-[perceivable content][] before the block but after the previous [perceivable content][], or any non-[perceivable content][] inside the [block][] but before the first [perceivable content][] in it.
```

Comment:
🤔 The intention was that it cannot (i.e. the last perceivable content before the block is not "at the start" since it could be, e.g., a huge `p` element and jumping to that element would not be perceived (by users) as jumping "at the start" of the block.

Agreed that current formulation is not clear enough on that.